HW_1/src/index.js

```diff
@@ -0,0 +1,16 @@
+import "./app.css";
+import React from "react";
+import ReactDOM from "react-dom";
+
+import UserForm from "./components/userForm";
+import Greetings from "./components/greetings";
+
+ReactDOM.render(<UserForm formHeader={onSubmit}/>, document.getElementById("root"));
```

Comment:
index.js shouldn't contain any screen-specific logic. It should be encapsulated in App component (or something like that)